test/support/factory.ex

```diff
@@ -44,8 +44,9 @@ defmodule Pairmotron.Factory do
 
   def group_factory do
     %Group{
-      name: sequence(:name, &"name #{&1}"),
-      owner: build(:user)
+      name: sequence(:name, &"group #{&1}"),
+      owner: build(:user),
+      users: []
```

Comment:
Although nothing was failing because this wasn't specified, this should make the factory a bit more robustness in the future.